src/package.json

```diff
@@ -28,8 +28,7 @@
     "node-fetch": "^3.3.1",
     "node-geocoder": "^4.2.0",
     "node-migrate": "^0.1.0",
-    "pdfkit": "^0.13.0",
-    "puppeteer": "^21.1.0",
+    "puppeteer": "16.2.0",
```

Comment:
this stinks, will keep trying to upgrade but need it to work for first user next week